ailab/benchmark/migrations/0001_initial.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-02-19 20:21
```

Comment:
I wonder whether we should check in this file, or users should generate this file themselves. 

I feel, if this file is generated, it should not be in the same directory as the repo. Is it possible to specify a place to put the generated files?

Or, at least, ignore this directory in git.

Reply:
Done. Deleted the migration file and added them to .gitignore.